pkgs/development/python-modules/imageio/default.nix

```diff
@@ -32,24 +31,19 @@
 
 buildPythonPackage rec {
   pname = "imageio";
-  version = "2.32.0";
+  version = "2.33.0";
   pyproject = true;
 
-  disabled = pythonOlder "3.7";
+  disabled = pythonOlder "3.8";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-5CWtNsYFMI2eptk+2nsJh5JgWbi4YiDhQqWZp5dRKN0=";
+  src = fetchFromGitHub {
+    owner = "imageio";
+    repo = "imageio";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-WoCycrJxo0vyV9LiWnEag1wbld3EJWu8mks8TnYt2+A=";
   };
```

Comment:
Was there any particular reason you switched to GitHub? To be clear, I think this is a perfectly reasonable change, I'm just wondering.

Reply:
> Was there any particular reason you switched to GitHub? To be clear, I think this is a perfectly reasonable change, I'm just wondering.

pypi tarballs are normally preprocessed, so not source codes in their natural habitat.